geom.js

```diff
@@ -3,6 +3,16 @@ class Rectangle {
     this.length = length;
     this.width = width;
   }
+  isSquare(){
+    this.length === this.width? true:false
```

Comment:
Don't forget a `return` statement here. That way you could use this method in situations like this...

```js
if(squareInstance.isSquare()){
  // code that would run if it's a square
}